content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -235,36 +318,45 @@ When you use `ILIKE`, the `%` wildcard character(s) should be included in the va
 ```ts
 const userName = 'Sarah'
 const emailFragment = 'prisma.io'
-const result = await prisma.$queryRaw(
-  'SELECT * FROM "User" WHERE (name = $1 OR email = $2)', // Using %$2 here would not work
+const result = await prisma.$queryRaw`
+  SELECT * FROM "User" WHERE (name = $1 OR email = $2), 
   userName,
-  `%${emailFragment}`
-)
+  %${emailFragment}`
 ```
 
+> **Note**: Using %$2 as an argument would not work
+
 #### <inlinecode>ALTER</inlinecode> limitation (PostgreSQL)
 
 PostgreSQL [does not support using `ALTER` in a prepared statement](https://www.postgresql.org/docs/current/sql-prepare.html), which means that the following queries will not work:
 
 ```ts
 await prisma.$executeRaw`ALTER USER prisma WITH PASSWORD '${password}'` // Tagged template
-await prisma.$executeRaw(`ALTER USER prisma WITH PASSWORD $1`, newPassword) // Parameterized query
+await prisma.$executeRaw`ALTER USER prisma WITH PASSWORD $1, newPassword` // Parameterized query
 ```
 
 You can use the following query to [work around the issue](https://github.com/prisma/prisma-client-js/issues/940#issuecomment-748014175), but be aware that this is potentially **unsafe** as `${password}` is not escaped:
```

Comment:
I think there might be an issue here since we say $executeRaw is the safe one and $executeRawUnsafe is the unsafe one. How could $executeRaw be unsafe?

Reply:
Shouldn't that then be done using the `Unsafe` signature?

Reply:
This section i wasnt sure about as it relates to an issue. I asked about this in slack yesterday too.